### PR TITLE
Add synapse

### PR DESCRIPTION
**Pull request by tigerza117**
this not work

### CODE REVIEW

**Comment by XFuryMCPE** [comment]
does it actually work? or u just copied the files

File: README.md [hunk 1]
```diff
@@ -15,3 +15,5 @@ Things you'll want to change on your plugins:
    - Players don't fall out of the world naturally, you'll want to handle PlayerMoveEvent as needed to kill them
 
 To build, run the server with DevTools installed then run /makeserver. It'll drop a phar file in it's plugin directory.
+
+Steadfast2 For MiniGmae Server Not Save Player
```
Comment (thebigsmileXD) [inline README.md]: MiniGame

Comment (xZeroMCPE) [inline README.md]: lol  "Steadfast2 For MiniGmae Server Not Save Player" 

It should be "SteadFast2 ~ For MiniGames server, does not save player files nor inventory dates". Or something similar.

**Comment by Sandertv** [comment]
Just copied, the Genisys thing is still there.

**Pull request closed without merging**